KOGITO-6740: [SW] Implement Switch state eventTimeout (#4184)
kiegroup/kie-docs: doc-content/kogito-docs/src/main/asciidoc/release-notes/ReleaseNotesKogito.1.19.0.Final/ReleaseNotesKogito.1.19.0.Final.adoc

== {PRODUCT} runtimes

> ===  Serverless Workflow updates for {PRODUCT}

Reviewer: ```suggestion
===  Serverless workflow updates for {PRODUCT}
```

> * Serverless workflow event timeouts support was implemented for the https://github.com/serverlessworkflow/specification/blob/0.8.x/specification.md#callback-state[Callback] and https://github.com/serverlessworkflow/specification/blob/0.8.x/specification.md#switch-state[Switch] states. Now, it is possible to define a waiting time for the event reception on those states. +

Reviewer: ```suggestion
* Serverless workflow event timeouts support is implemented for the https://github.com/serverlessworkflow/specification/blob/0.8.x/specification.md#callback-state[Callback] and https://github.com/serverlessworkflow/specification/blob/0.8.x/specification.md#switch-state[Switch] states. It is now possible to define a waiting time for the event reception in callback and switch states. 
```

> NOTE: For additional information about serverless workflow timeouts definition refer to the following serverless workflow specification section https://github.com/serverlessworkflow/specification/blob/0.8.x/specification.md#Workflow-Timeouts[Workflow Timeouts].

Reviewer: ```suggestion
NOTE: For more information about serverless workflow timeouts definition, see https://github.com/serverlessworkflow/specification/blob/0.8.x/specification.md#Workflow-Timeouts[Workflow Timeouts] in the serverless workflow specification section.
```
